test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Comment explaining what this file is for (what things people should/shouldn't add to it, that it's run on Travis, etc.)

Reply:
Added some comments.